presto-main/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -261,6 +264,15 @@ private URLClassLoader createClassLoader(List<Artifact> artifacts, String name)
             log.debug("    %s", file);
             urls.add(file.toURI().toURL());
         }
+
+        if (discoverPlugins) {
```

Comment:
move this to `buildClassLoaderFromCoordinates` since we only want this for dev mode